notebooks/summit/AuxTel/SITCOM-627/SITCOM-627_Measure_Astigmatism.ipynb

```diff
@@ -0,0 +1,1511 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    from lsst.ts.externalscripts.auxtel.latiss_cwfs_align import LatissCWFSAlign
</pre><p>You should use LatissWEPAlign. We're now moving away from LatissCWFSAlign.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
👍